client/app/store.ts

```diff
@@ -0,0 +1,24 @@
+import { configureStore } from '@reduxjs/toolkit'
+import { setupListeners } from '@reduxjs/toolkit/query';
+
+import authSliceReducer from '../features/authSlice';
+
+import {api} from '../features/api'
```

Comment:
nit: Usually api is the default export for the API slice, that way only the query and mutation hooks are required by destructuring

Reply:
OK i changed the bottom of the `api.tsx` file to be this: 
```
  export default api;

  export const { 
    useGetAllZoopsQuery, 
    useRegisterMutation 
  } = api
```

and then made line 6 in `store.ts` be `import api from '../features/api'`

Was that what you were wanting me to do?